src/main/java/org/example/utility/UtilityMethods.java

```diff
@@ -37,4 +37,15 @@ public static int countEven(int[] array) {
         }
         return evenElements;
     }
+
+    public static void reverse(int[] array) {
+        int temp;
```

Comment:
Please, check that the array is not null using `com.google.common.base.Preconditions.checkNotNull` method